packages/use-contractkit/__tests__/connectors/ledger.test.ts

```diff
@@ -0,0 +1,58 @@
+import { CeloContract } from '@celo/contractkit';
+
+import { Alfajores, localStorageKeys, WalletTypes } from '../../src';
```

Comment:
These imports could all be a lot better if we used jest's `moduleNameMapper`

```
moduleNameMapper: {
  "src": "<rootDir>/src/",
```

Reply:
im gonna open a separate issue for this since all our ts is also currently relative paths

Reply:
https://github.com/celo-org/use-contractkit/issues/199